lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -202,6 +202,37 @@ void setKernelArg(cl_kernel kernel, unsigned argIdx, T value) {
   GLOW_ASSERT(err == CL_SUCCESS && "Unable to set parameter");
 }
 
+template<class T>
+void setKernelArgShape(cl_kernel kernel, unsigned argIdx, T shapeValue) {
```

Comment:
Can you explain in the source code comment what is going on in this function and why? 

Are you trying to ensure a proper alignment?

Reply:
If this is to ensure the alignment or padding, is it required by the OpenCL spec? It would be interesting to have a reference then.

Reply:
Resolve a inssue sizeof(size_t) would be 4 on x86 platform(or armeabi)
And in OpenCL we defines ShapeNHWC ShapeNCHW PaddingTLBR ShapeHW to be tuple of
uint64_t, we need to make the parameter to be consistence.

Reply:
It's not about OpenCL spec.
But the types defined  in kernel.cl